version.txt

```diff
@@ -0,0 +1 @@
+0.2.0-rc1
```

Comment:
Having `rc1` suffix here is a bit out of place, but it depends on how the build works in ET.  Here is how we implement it on PyTorch and other libraries atm:

1. The version on `main` points to the next release, i.e. https://github.com/pytorch/pytorch/blob/main/version.txt is pointing to `2.4.0a0`.  This is used to built nightly binaries, i.e. https://download.pytorch.org/whl/nightly/cpu-cxx11-abi/torch-2.4.0.dev20240316%2Bcpu.cxx11.abi-cp310-cp310-linux_x86_64.whl
2. After doing branch cut, we bump the version, i.e. https://github.com/pytorch/pytorch/pull/121974
3. Realese binaries are built without the RC suffix, so the latest RC binaries will overwrite previous one, and the test  installation is always the same throughout the release process, i.e. `pip3 install torch torchvision torchaudio executorch==0.2.0 --index-url https://download.pytorch.org/whl/test/cpu`
4. Once the final RC is ready, the files under https://download.pytorch.org/whl/test/cpu will be published to pypi/conda/poetry.

So, in a sense, RC tag is used to triggered release build, but not included as part of the artifact name.